Samples/Windowing/cs-winui/SampleApp/WindowingInterop.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using Microsoft.UI;
+using Microsoft.UI.Windowing;
+using WinRT;
+
+namespace SampleApp
+{
+    [ComImport]
+    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    [Guid("EECDBF0E-BAE9-4CB6-A68E-9598E1CB57BB")]
+    internal interface IWindowNative
```

Comment:
We should use the new COM interop APIs here: https://docs.microsoft.com/windows/apps/desktop/modernize/winrt-com-interop-csharp